test/studies/arkouda/sub_test

```diff
@@ -38,6 +41,17 @@ else
   export "PATH=${ARKOUDA_HOME}/dep/hdf5-install/bin:$PATH"
 fi
 
+# CHPL_TEST_ARKOUDA_DISABLE_MODULES is a colon separated list of modules to
+# disable.  Disable these modules by commenting them out in ServerModules.cfg
+if [ ! -z "${CHPL_TEST_ARKOUDA_DISABLE_MODULES+x}" ] ; then
```

Comment:
What's the `+x` do here? To me it seems like we enter this branch always because of it, and it seems like we'd want to skip this is  `CHPL_TEST_ARKOUDA_DISABLE_MODULES` is the empty string

Reply:
This also seems mildly fragile in that nightly testing will fail if a new module is added that doesn't work with GPUs. An alternative might be to create your own file that's a snapshot of what you want to today and set `ARKOUDA_CONFIG_FILE` to point to that for the GPU configuration.

Reply:
I more or less copied it from here: https://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash

The `+x` thing is so the test passes if `$CHPL_TEST_ARKOUDA_DISABLE_MODULES` is defined as the empty string. You're right in this case it doesn't really matter (there's no processing I need to do with the empty string anyway).

And I should just change the conditional to `-n` anyway so it reads clearer.

Reply:
> This also seems mildly fragile in that nightly testing will fail if a new module is added that doesn't work with GPUs. An alternative might be to create your own file that's a snapshot of what you want to today and set ARKOUDA_CONFIG_FILE to point to that for the GPU configuration.

I think I would want it to fail in this case though. A new module that's incompatible (potentially revealing a previously unknown bug) is exactly the sort of thing I'd want to be informed about.

Reply:
Yeah, makes sense on being informed. I think `-n` is currently triggering to because you default `CHPL_TEST_ARKOUDA_DISABLE_MODULES` to ` ` (single space.) Probably just don't set that env var in this script now?